GPU/Math3D.h

```diff
@@ -634,6 +634,13 @@ class Vec4
 		return Vec4(VecClamp(x, l, h), VecClamp(y, l, h), VecClamp(z, l, h), VecClamp(w, l, h));
 	}
 
+	Vec4 Reciprocal() const
+	{
+		// In case we use doubles, maintain accuracy.
```

Comment:
Sorry, was worrying about the accuracy problems and trying to mess with values to fix things - removed the comment.

-[Unknown]